modules/virt-disabling-tls-for-registry.adoc

```diff
@@ -14,8 +14,14 @@ You can disable TLS (transport layer security) for one or more container registr
 
 .Procedure
 
-* Edit the `HyperConverged` custom resource and add a list of insecure registries to the `spec.storageImport.insecureRegistries` field.
+* Use the following command to edit the `HyperConverged` custom resource and add a list of insecure registries to the `spec.storageImport.insecureRegistries` field.
 +
+[source,terminal]
+----
+$ oc edit Hyperconverged kubevirt-hyperconverged -n {CNVNamespace}
+----
++
+.Example `HyperConverged` custom resource
```

Comment:
**NOTE TO REVIEWER** The `-n {CNVNamespace}` is used throughout the Virtualization book. I didn't update here; would likely be best as a separate PR, if required.

Reply:
This should work with `subs="attributes+"`; I'll make a note to look for instances that are missing this